src/map/skill.cpp

```diff
@@ -7630,6 +7630,29 @@ int skill_castend_nodamage_id (struct block_list *src, struct block_list *bl, ui
 
 		break;
 
+	case NPC_MOVE_COORDINATE:
+		{
+			clif_skill_nodamage(src, bl, skill_id, skill_lv, 1);
+			clif_skill_damage(src, bl, tick, status_get_amotion(src), 0, -30000, 1, skill_id, skill_lv, DMG_SINGLE);
```

Comment:
Those should be after `skill_check_unit_movepos`, the skill animation shouldn't be shown when the skill fails

Reply:
I did like this because the skill animation would not be located proper place.

Reply:
![move_coordinate](https://user-images.githubusercontent.com/47050704/224581819-a81cf947-2e4d-4079-8572-282ef932b86f.gif)

Check this from KRo, that the effects created before units moved.